$NetBSD: patch-configure,v 1.4 2023/01/10 22:07:33 hauke Exp $

Fix build on SunOS by disabling the isainfo bitness check 

--- configure.orig	2023-01-10 08:57:35.000000000 +0000
+++ configure
@@ -19331,7 +19331,7 @@ printf %s "checking if we have to build 
 	   # check for isainfo, if not found it has to be a 32 bit kernel (<=2.6)
 	   if test -x /usr/bin/isainfo; then
 		# check for 64 bit platform
-		if isainfo -kv | grep '^64-bit'; then
+		if test "$MACHINE_ARCH" = "x86_64"; then
 			COMPILE_64BIT_KMODULE=yes
 		fi
 	   fi
